pkgs/development/python-modules/stack-data/default.nix

```
@@ -0,0 +1,57 @@
+{ asttokens
+, buildPythonPackage
+, executing
+, fetchgit
+, git
+, lib
+, littleutils
+, pure-eval
+, pygments
+, pytest
+, setuptools_scm
+, toml
+, typeguard
+}:
+
+buildPythonPackage rec {
+  pname = "stack_data";
+  version = "0.0.7";
+
+  # Install requires .git and fetchFromGitHub doesn't support leaving it so
+  # let's use fetchgit
+  src = fetchgit {
+    url = "https://github.com/alexmojaki/${pname}.git";
+    rev = "v${version}";
+    sha256 = "0smifhq8ma8qqhsawmgbfxq5i07rszgysf23i38dlal4h9krm03z";
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    git
+  ];
+
+  buildInputs = [
+    setuptools_scm
+    toml
+  ];
+
+  propagatedBuildInputs = [
+    asttokens
+    executing
+    pure-eval
+  ];
+
+  checkInputs = [
+    littleutils
+    pygments
+    pytest
```

```suggestion
    pytestCheckHook
```
tests aren't actually being ran
```
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
Finished executing setuptoolsCheckPhase
pytestcachePhase
/nix/store/v12flc8l02vg93dynbyrp46q99bks7fd-python3.8-stack_data-0.0.7
```

---

Oh, now I learned something new! :+1: Yes, fixed now.